cg/meta/demultiplex/utils.py

```diff
@@ -206,11 +209,18 @@ def parse_manifest_file(manifest_file: Path) -> list[Path]:
 
 def is_file_relevant_for_demultiplexing(file: Path) -> bool:
     """Returns whether a file is relevant for demultiplexing."""
-    relevant_directories = [DemultiplexingDirsAndFiles.INTER_OP, DemultiplexingDirsAndFiles.DATA]
-    for relevant_directory in relevant_directories:
-        if relevant_directory in file.parts:
-            return True
-    return False
+    irrelevant_directories = [
```

Comment:
should this be in the constants module?

Reply:
I'm really not sure. They are, per definition, directories we don't care about so we won't use them anywhere else 😅